GliaWidgets/Sources/ViewModel/Call/Data/Call.swift

```diff
@@ -234,27 +234,30 @@ class Call {
     private func updateMediaStream<Streamable>(_ mediaStream: ObservableValue<MediaStream<Streamable>>,
                                                with stream: Streamable,
                                                isRemote: Bool) {
+        let mediaStreamValue: MediaStream<Streamable>
+        defer {
```

Comment:
Interesting, do you know why this fixes it?

Reply:
My only assumption, that this is timing issue for `ObservableValue`. First I discovered "fix" by putting break point that does not suspend execution, but continues evaluation and plays sound - that was enough to get video to show up, then I tried famous `DispatchQueue.main.async` - that also worked, then at last - `defer`.

Reply:
Looks not very reliable. Perhaps would be good to make a build by `mater-build` Bitrise workflow and ask other to verify this fix.
Wdyt?

Reply:
@EgorovEI could you please assist me with that?